etc/kayobe/environments/ci-aio/stackhpc-ci.yml

```diff
@@ -8,9 +8,10 @@ kolla_docker_namespace: stackhpc-dev
 ###############################################################################
 # Network configuration.
 
-# Don't touch resolv.conf: use Neutron DNS for accessing Pulp server via
-# hostname.
```

Comment:
I think we don't actually make use of this - we write out an entry in /etc/hosts. I suggest we just remove all of this config.

Reply:
To clarify do you mean remove resolv config entirely or just remove the hostname resolv_domain?

Reply:
I'll close this one and make a pr.